Tasks/ANT/ant.ps1

```diff
@@ -38,6 +42,27 @@ if($jdkVersion -and $jdkVersion -ne "default")
 Write-Verbose "Running Ant..."
 Invoke-Ant -AntBuildFile $antBuildFile -Options $options -Targets $targets
 
+# Publish test results files
+$publishJUnitResultsFromAntBuild = Convert-String $publishJUnitResults Boolean
+if($publishJUnitResultsFromAntBuild)
+{
+   # check for JUnit test result files
+    $matchingTestResultsFiles = Find-Files -SearchPattern $testResultsFiles
+    if (!$matchingTestResultsFiles)
+    {
+        Write-Host "No JUnit test results files were found matching pattern `$testResultsFiles`, so publishing JUnit test results is being skipped."
+    }
+    else
+    {
+        Write-Verbose "Calling Publish-TestResults"
+        Publish-TestResults -TestRunner "JUnit" -TestResultsFiles $matchingTestResultsFiles -Context $distributedTaskContext
+    }    
+}
+else
+{
+    Write-Host "Option to publish JUnit Test results produced by Ant build was not selected and is being skipped."
```

Comment:
okay, it just seems odd to print the statement when the user set the option "publishJUnitResults" to false.